src/ds/ptrwrap.h

```diff
@@ -127,23 +196,21 @@ namespace snmalloc
 #  pragma warning(push)
 #  pragma warning(disable : 4702)
 #endif
-    constexpr explicit CapPtr(uintptr_t p) : unsafe_capptr(p) {}
+    explicit CapPtr(T* p) : unsafe_capptr(p) {}
```

Comment:
Can this not be constexpr without the type casting?

Reply:
Good call; probably just forgot to put the keyword back during some churn.